packages/eds-data-grid-react/src/EdsDataGrid.tsx

```diff
@@ -227,6 +228,7 @@ export function EdsDataGrid<T>({
     debugHeaders: debug,
     debugColumns: debug,
     enableRowSelection: rowSelection ?? false,
+    enableMultiRowSelection: multiRowSelection,
```

Comment:
I would recommend using the same property name as the library.

Reply:
@torleifhalseth The `enable` prefix was added to make it clearer what the prop does. I'd argue that prefixing `enableMultiRowSelection` as well makes sense, given this.

Reply:
have we concluded on this? Is there any further work needed or can i merge this?

Reply:
Imo we can leave it as is, since it matches the pattern for enableRowselection already in place.